src/coreclr/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -19,6 +19,7 @@ MetaDataGetDispenser
 mono_unity_initialize_host_apis
 coreclr_array_length
 coreclr_class_from_systemtypeinstance
+coreclr_image_get_custom_attribute_data
```

Comment:
These exports need kept in sync between the Windows and *nix files. I would expect the same changes in both.

Reply:
are there other files than `src/coreclr/dlls/mscoree/mscorwks_unixexports.src` and `src/coreclr/dlls/mscoree/mscorwks_ntdef.src`? I've added new api to both files and synchronized missing exports

Reply:
Those are the only files AFAIK.